lib/OCP/TalkBackend.php

```diff
@@ -15,13 +16,16 @@
 class TalkBackend implements ITalkBackend {
 	protected Manager $manager;
 	protected ParticipantService $participantService;
+	private RoomService $roomService;
```

Comment:
```suggestion
	protected RoomService $roomService;
```

Reply:
As a app developer since ages and phpbb extension developer before that I really dislike the use of private. It really often killed possible extensions and had to wait for an upstream fix (which you can barely version depend on) instead of just extending an existing implementation.